indexing/src/test/java/com/google/enterprise/cloudsearch/sdk/indexing/IndexingServiceTest.java

```diff
@@ -731,6 +732,19 @@ public void testUpdateItemWithContentHash() throws IOException {
     verify(quotaServer, times(1)).acquire(Operations.DEFAULT);
   }
 
+  @Test
+  public void testUpdateItemWithDefaultQueue() throws GeneralSecurityException, IOException {
```

Comment:
nit, just throw Exception for tests (instead of all named individual exception).

Reply:
I was just copying existing test cases ; )